internal/testplanet/uplink.go

```diff
@@ -39,7 +39,7 @@ type Uplink struct {
 	Transport        transport.Client
 	StorageNodeCount int
 
-	APIKey    map[storj.NodeID]string
+	APIKey    map[storj.NodeID]*macaroon.APIKey
```

Comment:
If this had a ref to `uplink.APIKey` then all the:

```
apikey, err := uplink.ParseAPIKey(planet.Uplinks[0].APIKey[satellite.ID()].Serialize())
```
could be replaced with:

```
apikey := planet.Uplinks[0].APIKey[satellite.ID()]
```

Reply:
heh, i don't know why i didn't think of that

Reply:
okay it turns out this sucks either way. you can't create a libuplink.APIKey, you can only parse one. there's new libuplink.NewAPIKey method, but there is a macaroon.NewAPIKey method, so no matter what somewhere we have parsing/serializing to convert (without adding otherwise unnecessary public functions)

Reply:
Oh well then.